port/unix/omrsysinfo.c

```diff
@@ -7687,8 +7687,104 @@ omrsysinfo_get_processes(struct OMRPortLibrary *portLibrary, OMRProcessInfoCallb
 	portLibrary->mem_free_memory(portLibrary, command);
 	closedir(dir);
 	return callback_result;
-#else /* defined(LINUX) */
+#elif defined(AIXPPC) /* defined(LINUX) */
+#define MAX_PROCS 256
```

Comment:
Please move the addition before LINUX (so platform support is ordered alphabetically).